python/determineLuminosity.py

```diff
@@ -248,14 +252,16 @@ def simulateDataOnHimster(
 
                     #! these mus be applied again, because they can be overridden at command time
                     # (the config on disk doesn't change and doesn't know this)
+                    rec_par.num_events_per_sample=box_num_events_per_sample
+                    rec_par.num_samples=box_num_samples
                     rec_par.use_xy_cut = thisScenario.use_xy_cut
                     rec_par.use_m_cut = thisScenario.use_m_cut
 
-                    rec_par.reco_ip_offset = (
-                        ip_info_dict["ip_offset_x"],
-                        ip_info_dict["ip_offset_y"],
-                        ip_info_dict["ip_offset_z"],
-                    )
+                    #rec_par.reco_ip_offset = (
+                    #    ip_info_dict["ip_offset_x"],
+                    #    ip_info_dict["ip_offset_y"],
+                    #    ip_info_dict["ip_offset_z"],
+                    #)
```

Comment:
The reci ip dict is changed [here](https://github.com/panda-luminosity-detector-group/LuminosityFit/blob/cad5c87b0cb87ce177cbe42d8b2bf52ae72b7a4b/python/determineLuminosity.py#L209-L212) after a first reconstruction run. At this point, the IP may have changed but that's not known to the experimentConfig yet. Maybe we should write that to a log somewhere. I would prefer not to change the config file itself, because that is supposed to be the known start config.

Reply:
The ip_offset_z is always 0 if we use line 260-264, so It's better we do not use them, just use the experimentConfig.

Reply:
For z yes, but that parameter widens the theta search angle if the IP is not at 0.

Reply:
So maybe we can just delete line 263.